source/docs/software/dashboards/index.rst

```diff
@@ -4,10 +4,7 @@
    :maxdepth: 1
 
    dashboard-intro
-   shuffleboard/index
```

Comment:
I think not having things referenced in an index causes a build failure. Since they will eventually be stubs, probably just move them to the end rather then deleting